jcl/src/java.management/share/classes/java/lang/management/DefaultPlatformMBeanProvider.java

```diff
@@ -61,9 +67,21 @@
 			.addInterface(java.lang.management.OperatingSystemMXBean.class)
 			.register(allComponents);
 
-		ComponentBuilder.create(LoggingMXBeanImpl.getInstance())
-			.addInterface(java.lang.management.PlatformLoggingMXBean.class)
-			.register(allComponents);
+		try {
+/*[IF Sidecar19-SE-B165]*/
+			ModuleLayer.boot().findModule("java.logging").get();
+/*[ELSE]
+			Layer.boot().findModule("java.logging").get();
+/*[ENDIF]*/
+		
+			ComponentBuilder.create(LoggingMXBeanImpl.getInstance())
+				.addInterface(java.lang.management.PlatformLoggingMXBean.class)
+				.register(allComponents);
+		} catch (Exception e) {
```

Comment:
What is the exception that occurs when logging is not available? Can this catch something more specific than Exception?

Reply:
Seems this code could test if logging is available using isPresent​() rather than causing an exception by calling get().